Libraries/plist/Tests/Format/test_JSON.cpp

```diff
@@ -98,3 +98,38 @@ TEST(JSON, Empty)
     ASSERT_EQ(deserialize.first, nullptr);
 }
 
+TEST(JSON, SingleQuotes)
+{
+    /* Single quotes are not allowed in JSON. */
+    auto contents = Contents("{ 'one': 1 }");
+    auto deserialize = JSON::Deserialize(contents, JSON::Create());
+    EXPECT_EQ(deserialize.first, nullptr);
+}
+
+TEST(JSON, Number)
```

Comment:
Turns out I can't test the numbers because this is testing the cases where decoding fails so there's no number to check. :P